src/display/api_compatibility.js

```diff
@@ -28,18 +28,18 @@ if (typeof PDFJSDev === "undefined" || PDFJSDev.test("GENERIC")) {
     // Sometimes IE and Chrome on iOS losing the data created with
     // createObjectURL(), see issues #3977 and #8081.
     if (isIE || isIOSChrome) {
-      compatibilityParams.disableCreateObjectURL = true;
+      apiCompatibilityParams.disableCreateObjectURL = true;
     }
   })();
 
   // Support: Node.js
   (function checkFontFaceAndImage() {
     // Node.js is missing native support for `@font-face` and `Image`.
     if (isNodeJS) {
-      compatibilityParams.disableFontFace = true;
-      compatibilityParams.nativeImageDecoderSupport = "none";
+      apiCompatibilityParams.disableFontFace = true;
+      apiCompatibilityParams.nativeImageDecoderSupport = "none";
     }
   })();
 }
 
-exports.apiCompatibilityParams = Object.freeze(compatibilityParams);
```

Comment:
Do you happen to know why we had `Object.freeze` here and in the viewer file in the first place? I can only imagine that it should not be mutated later on, but AFAICT we never do that anyway.

Reply:
> Do you happen to know why we had `Object.freeze` here and in the viewer file in the first place?

Yes, to avoid the object being mutated by accident since you can still add/remove/change properties even in a `const` object.
The better question is why I changed this here, to which there's no good answer; I'll revert these changes, well spotted!